packages/Tag/src/Tag.js

```diff
@@ -17,7 +17,13 @@ export default function Tag(props) {
       {avatar}
       <sc.Ellipsis size={size}>{children}</sc.Ellipsis>
       {onRemove ? (
-        <sc.Delete isDisabled={isDisabled} size={size} data-pka-anchor="tag.remove" onClick={handleRemove} />
+        <sc.Delete
+          isDisabled={isDisabled}
+          size={size}
+          data-pka-anchor="tag.remove"
+          data-qa-anchor="tag.remove"
```

Comment:
Was this the naming convention we settled on? I thought there was a `paprika` namespace or something?

Reply:
Oh yep, i'll add `paprika.` as a prefix